docs/view-default.md

```diff
@@ -204,7 +206,7 @@ that allows Orb to position the nodes.
 ![](./assets/view-default-fixed.png)
 
 ```typescript
-import { DefaultView } from '@memgraph/orb';
+import { DefaultView } from "@memgraph/orb";
```

Comment:
We have to align our formatters at some point. Seems like every time we open a markdown file and save it we have different outputs.